src/mito2/src/test_util.rs

```diff
@@ -67,26 +68,29 @@ pub(crate) fn new_noop_file_purger() -> FilePurgerRef {
 }
 
 /// Env to test mito engine.
+#[cfg(test)]
```

Comment:
question: Why do we still need to add `#[cfg(test)]` here? It is caused by the `testing` feature required by the ObjectStoreManager?

If so, I think we should remove the `testing` feature from `ObjectStoreManager::from_default()`

Reply:
> Why do we still need to add #[cfg(test)] here?

Sorry. This is my mistake. We don't need `#[cfg(test)]` at all😅